src/components/Card.js

```diff
@@ -25,16 +59,17 @@ export default class Card {
       });
   }
 
-  _handleLikeIcon() {
-    this._cardElement
-      .querySelector(".card__like-button")
-      .classList.toggle("card__like-button_active");
-  }
-  _handleDeleteButton() {
+  deleteCard() {
     this._cardElement.remove();
```

Comment:
There is an error in the console when you removing a card ` index.js:55 TypeError: Cannot read properties of null (reading 'remove')
    at Card.deleteCard (Card.js:63:23)
    at index.js:51:27`